docs/source/exposing.md

```diff
@@ -0,0 +1,267 @@
+
+### Node Service Template
+
+nodeService serves as a template for a node-dedicated Service deployed by the Scylla Operator for each node within a ScyllaCluster. 
+The properties you receive depend on the type you select. 
+Additionally, you have the option to define custom annotations, which will be incorporated into each node Service, 
+these might be useful to further tweak Service resources.
+
+#### `Headless` Type
+
+For `Headless` type, Scylla Operator creates a Headless Service with selector pointing to particular node in the ScyllaCluster.
+Such Service doesn't provide any additional IP address, and internal DNS records points directly to PodIP of a node.
+
+This type of Service is useful when ScyllaCluster nodes broadcast PodIPs to clients or other nodes.
+
+Example:
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+spec:
+  exposeOptions:
+    nodeService:
+     type: Headless
+```
+
+#### `ClusterIP` Type
+
+For `ClusterIP` type, Scylla Operator creates a ClusterIP Service pointing to particular node in the ScyllaCluster.
+ClusterIP Service assigns an IP address from a pool of IP addresses that your Kubernetes cluster has reserved for that purpose.
+
+These IP addresses are only routable within the same Kubernetes cluster, so it's a good fit, if you don't want to expose it to other networks.
+
+Example:
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+spec:
+  exposeOptions:
+    nodeService:
+     type: ClusterIP
+```
+
+#### `LoadBalancer` Type
+
+For the 'LoadBalancer' type, the Scylla Operator generates a LoadBalancer Service that directs traffic to a specific node within the ScyllaCluster. 
+On platforms with support for external load balancers, this Service provisions one. 
+The accessibility of this load balancer's address depends on the platform and any customizations made; in some cases it may be reachable from the internal network or public Internet.
+
+LoadBalancer Services are a superset of ClusterIP Services, implying that each LoadBalancer Service also possesses an allocated ClusterIP. 
+These LoadBalancer Services can be configured using the following fields, and these configurations are mirrored across every node Service:
+* externalTrafficPolicy
+* internalTrafficPolicy
+* loadBalancerClass
+* allocateLoadBalancerNodePorts
+
+Check [Kubernetes Service documentation](https://kubernetes.io/docs/concepts/services-networking/service), to know more about these options.
+
+Example:
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+spec:
+  exposeOptions:
+    nodeService:
+     type: LoadBalancer
+     loadBalancerClass: my-custom-load-balancer-class
+```
+
+---
+
+### Broadcast Options
+
+Broadcast options controls what is the source of the address being broadcasted to other clients and nodes.
+It's configured independently for clients and nodes because you may want to expose these two types of traffic on different networks, due to costs or security concerns.
```

Comment:
```suggestion
It's configured independently for clients and nodes because you may want to expose these two types of traffic on different networks. Using different networks can help manage costs, reliability, latency, security policies or other metrics you care about.
```

Reply:
done